README.md

```diff
@@ -26,7 +26,7 @@ Install it via composer
 composer install redcode/tree-bundle
```

Comment:
Nice suggestion!
@RomainSanchez please have a look and I will merge these changes